[master] modules.rabbitmq: In v3.7.9 list_* commands keep column names. To sup…

terentev-mn opened the pull request:
…ress it need use '--no-table-headers'

https://github.com/rabbitmq/rabbitmq-cli/issues/273

### What does this PR do?

### What issues does this PR fix or reference?
Fixes:
  list_users, list_vhosts, list_parameters, list_queues, list_policies methods

### Previous Behavior
output always contents colunm name

### New Behavior
supress colunm name

### Merge requirements satisfied?
**[NOTICE] Bug fixes or features added to Salt require tests.**
<!-- Please review the [test documentation](https://docs.saltproject.io/en/master/topics/tutorials/writing_tests.html) for details on how to implement tests into Salt's test suite. -->
- [ ] Docs
- [ ] Changelog - https://docs.saltproject.io/en/master/topics/development/changelog.html
- [ ] Tests written/updated

### Commits signed with GPG?
Yes/No

Please review [Salt's Contributing Guide](https://docs.saltproject.io/en/master/topics/development/contributing.html) for best practices.

See GitHub's [page on GPG signing](https://help.github.com/articles/signing-commits-using-gpg/) for more information about signing commits with GPG.

Commit:
modules.rabbitmq: In v3.7.9 list_* commands keep column names. To supress it need use '--no-table-headers'
https://github.com/rabbitmq/rabbitmq-cli/issues/273

twangboy reviewed (changes requested):
This looks great. Please create a changelog and a test.

dwoz committed:
Add changelog and tests for rabbitmq --no-table-headers fix

Add changelog/65520.fixed.md and six unit tests asserting that
list_users, list_vhosts, list_upstreams, list_queues,
list_queues_vhost, and list_policies each pass --no-table-headers
to rabbitmqctl, as requested by twangboy in PR review.
